src/chainstate/stacks/address.rs

```diff
@@ -30,32 +30,31 @@ use burnchains::bitcoin::address::{
 };
 use burnchains::{Address, BurnchainSigner, PublicKey};
 use chainstate::stacks::StacksPublicKey;
+use chainstate::stacks::{
+    C32_ADDRESS_VERSION_MAINNET_MULTISIG, C32_ADDRESS_VERSION_MAINNET_SINGLESIG,
+    C32_ADDRESS_VERSION_TESTNET_MULTISIG, C32_ADDRESS_VERSION_TESTNET_SINGLESIG,
```

Comment:
Should we move this constants to the address module?

Reply:
I think it's ok for these to remain in `chainstate` so long as `StacksAddress` is there.

Reply:
Should StacksAddress be in types?

Reply:
Yes, `StacksAddress` should definitely be in `types`. These constants should probably live there too. Will make those changes in a future PR